brvs-core/src/main/java/iroha/validation/transactions/provider/impl/IrohaHelper.java

```diff
@@ -78,15 +90,48 @@ public IrohaHelper(
    *
    * @return {@link Observable} of Iroha proto {@link QryResponses.BlockQueryResponse} block
    */
-  Observable<BlockOuterClass.Block> getBlockStreaming() {
-    Queries.BlocksQuery query = new BlocksQueryBuilder(accountId, Instant.now(), 1)
-        .buildSigned(keyPair);
-    return irohaAPI.blocksQuery(query).map(response -> {
-          logger.info(
-              "New Iroha block arrived. Height " + response.getBlockResponse().getBlock().getBlockV1()
-                  .getPayload().getHeight());
-          return response.getBlockResponse().getBlock();
-        }
-    );
+  public Observable<BlockOuterClass.Block> getBlockStreaming() {
+    ConnectionFactory factory = new ConnectionFactory();
+    factory.setHost(RMQHost);
+    factory.setPort(RMQPort);
+    Connection conn = null;
+    Channel ch = null;
+    String queue = null;
+    try {
+      conn = factory.newConnection();
```

Comment:
There is `AutorecoveringConnection` as well. Should we use it instead of `AMQConnection` ?